### PR TITLE
Fix malicious package, update nodemon -> 1.18.7

### PR DESCRIPTION
**Pull request by mayurkale22**
The tests failed on CircleCI ~5 days, because of an event-stream incident. 

**NPM blog on malicious package:** https://blog.npmjs.org/post/180565383195/details-about-the-event-stream-incident.

nodemon@1.18.7: Completely removes the old method of `pstree[.remy]` and drops the event-stream vulnerability at the same time.

### CODE REVIEW

File: packages/opencensus-exporter-ocagent/package-lock.json [hunk 1]
```diff
@@ -94,12 +96,14 @@
     "@types/mocha": {
       "version": "2.2.48",
       "resolved": "https://registry.npmjs.org/@types/mocha/-/mocha-2.2.48.tgz",
-      "integrity": "sha512-nlK/iyETgafGli8Zh9zJVCTicvU3iajSkRwOh3Hhiva598CMqNJ4NcVCGMTGKpGpTYj/9R8RLzS9NAykSSCqGw=="
+      "integrity": "sha512-nlK/iyETgafGli8Zh9zJVCTicvU3iajSkRwOh3Hhiva598CMqNJ4NcVCGMTGKpGpTYj/9R8RLzS9NAykSSCqGw==",
+      "dev": true
```
Comment (bogdandrutu) [inline packages/opencensus-exporter-ocagent/package-lock.json]: Why did you add dev: true everywhere?

Comment (mayurkale22) [reply packages/opencensus-exporter-ocagent/package-lock.json]: package-lock.json is automatically generated. If ```"dev": true``` then this dependency is either a development dependency ONLY of the top level module or a transitive dependency of one. (Source: https://docs.npmjs.com/files/package-lock.json#dev)
